etc/env/base/vm_runner.toml

```diff
@@ -0,0 +1,9 @@
+# Configuration for the VM runner crate
```

Comment:
please update yaml config as well (etc/env/file_based)

Reply:
I think I have added it but I am struggling to find a way to test it locally.

Also, I tried to figure out the scheme in which way the naming hierarchy is structured (i.e. if `protective_reads_writer.protective_reads_db_path` is the correct path). My current guess is that it's based on the field names in `GeneralConfig` but I haven't found any relevant logic in the code.

Reply:
Nevermind, talked to @Deniallugo and apparently it's just based off of protobuf